Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewCell.cs

```diff
@@ -12,6 +12,13 @@ protected ItemsViewCell(CGRect frame) : base(frame)
 		{
 			CollectionView.VerifyCollectionViewFlagEnabled(nameof(ItemsViewCell));
 			ContentView.BackgroundColor = UIColor.Clear;
+
+			var selectedBackgroundView = new UIView
+			{
+				BackgroundColor = UIColor.Gray
+			};
+
+			SelectedBackgroundView = selectedBackgroundView;
```

Comment:
Android already handles it (default platform-normal selection handling, with the option to override via the VSM). 
It's already on my TODO list, but here you go in case I get hit by a bus: https://github.com/xamarin/Xamarin.Forms/issues/5050

Reply:
I was more thinking that you might move on to bigger, better things and forget about these lowly "issues". But, hey, go big bus, or take it home, right?